### ISSUE TITLE
Adding SVG Support

### ISSUE THREAD

**Issue by dougellison** [webcomponents/webcomponentsjs#419]
I've looked through the code and it clearly states there is no support for extending SVG elements.  I've done some simple experimentation on IE 11 and Firefox and it appears that if you do extend a SVG element and it is part of the DOM such as

```
<circle is="extended-circle"></circle>
```

It renders and works as expected.  However when trying to create extended SVG elements using the document.createElementNS it fails because there is no support.  

I'd like to know if any known reason why SVG element support cannot be added?  If there is no clear reasoning other then it was not added due to time it is something I am going to attempt.

**Comment by warpech** [comment]
I have also noticed a class of issues related to importing SVG nodes to DOM when polyfill is loaded.

Specifically problems include:
- using DOMParser on loaded XML
- DOM methods (`replaceChild` to be specific).

In the following Polymer Element (Polymer doesn't seem to matter here):
http://juicy.github.io/juicy-svg-injector/
http://github.com/juicy/juicy-svg-injector

I needed to make 2 changes to otherwise great SVGInjector library (v 1.1.3). These changes are marked in [SVGInjector.js](https://github.com/Juicy/juicy-svg-injector/blob/gh-pages/vendor/svg-injector.js) with comment "warpech". The original file is [here](https://github.com/iconic/SVGInjector/blob/master/svg-injector.js).

I would love to use original SVGInjector file when these problems are fixed in webcomponents.js polyfill.

To anyone investigating this, I recommend trying to use SVGInjector as indication of XML/SVG import working correctly.
